### PR TITLE
FakeHTable requires a valid configuration object.

### PR DESCRIPTION
**Pull request by kryzthov**

### CODE REVIEW

**Comment by kryzthov** [comment]
@lsheng Can you take a look?

File: src/main/scala/org/kiji/testing/fakehtable/FakeHBase.scala [hunk 1]
```diff
@@ -23,20 +23,20 @@ import java.util.Arrays
 import java.util.{List => JList}
 import java.util.{TreeMap => JTreeMap}
 
-import scala.collection.JavaConverters._
+import scala.collection.JavaConverters.asScalaIteratorConverter
 import scala.collection.mutable.Buffer
-import scala.Fractional
+import scala.math.BigInt.int2bigInt
```
Comment (lsheng) [inline src/main/scala/org/kiji/testing/fakehtable/FakeHBase.scala:28]: Is this being used for anything?  Weird that our scala checkstyle doesn't check for unused imports(or maybe that isn't possible).

Comment (kryzthov) [inline src/main/scala/org/kiji/testing/fakehtable/FakeHBase.scala:28]: As discussed offline, this can be removed.

**Comment by lsheng** [comment]
https://github.com/kijiproject/fake-hbase/commit/eff04766319ab9a6595e0b2f0b35b2447e676083

**Pull request closed without merging**